app/views/main_page/home.html.haml

```diff
@@ -13,19 +13,25 @@
         .left-nav
           Canteen Koala
         .right-nav
-          %div Home
+          .page-button= link_to 'Home', '#'
```

Comment:
it's a better practice to put the class on a separate line, like:

```
.page-button
  = link_to 'Home', '#'
```

Can you update this everywhere? :-)